<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotographerScreen</class>
 <widget class="QMainWindow" name="PhotographerScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно фотографа</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="WorkDoneBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>470</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Работа готова</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AcceptClientBtn">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>470</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Принять клиента</string>
    </property>
   </widget>
   <widget class="QTableView" name="WorksTableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>591</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EditCheckBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>470</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать квитанцию</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Квитанция №:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CheckNumberLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>71293140931</string>
    </property>
   </widget>
   <widget class="QLabel" name="CheckDateLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>31.12.19</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CheckTimeLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>15:45</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Список работ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кассир: </string>
    </property>
   </widget>
   <widget class="QLabel" name="CashierNameLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetNextCheckInQueueBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить первого в очереди</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SkipCheck">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Пропустить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователь</string>
    </property>
    <addaction name="ChangeUserAction"/>
    <addaction name="ExitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ChangeUserAction">
   <property name="text">
    <string>Сменить пользователя</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
